__tests__/ClassNames-test.coffee

```diff
@@ -27,16 +27,19 @@ Decorator = require '../Decorator.coffee'
 classTree =
   className: 'my-base-class'
   block: 'base'
+  _qaClassName: 'test-base'
```

Comment:
@istvan-hevele let's create a separate test suite and a test component for qa classes